### DISCUSSION TITLE
Deleting audio via the Recordings tab

### DISCUSSION THREAD

**Discussion by blogdemoi** [mcguirepr89/BirdNET-Pi#554] [Q&A] [answered]
Hello,

When I delete an audio recording in the "Recordings' tab, does it delete the event in the bird.db? Today I had a Strix aluco falsely detected, I deleted the audio in the "Recordings' tab, but it still showed in the "Recordings" tab for today (although no audio appeared).

Do I have to delete the event in the database too?

Thank you.

**Comment by ehpersonal38** [comment, marked as answer]
Yes it deletes both the DB entry and the audio file. I just tested it on my installation and all is working as expected. Maybe your database was busy for the moment you tried deleting? Can you try on a few more recordings and see if it works then?

**Reply by blogdemoi** [reply]
Thank you. I tried agin and it is working fine.
